### DISCUSSION TITLE
Width control of mesh lines

### DISCUSSION THREAD

**Discussion by sajutabraham** [neperfepx/neper#519] [Q&A] [answered]
@rquey  Sir, how to adjust the width of mesh lines in 2D and 3D mesh images? Is it using **-dataelt1drad** and **-dataelt3dedgerad** values? Can you please explain.

**Comment by rquey** [comment, marked as answer]
This is explained in the documentation, [here](https://neper.info/doc/neper_v.html#cmdoption-data-elt-0-1-d-node-elt-2-3-dedge-elset-0-1-d-rad): `-dataelt2dedgerad` for 2D elements, `-dataelt3dedgerad` for 3D elements, and `-dataelt1drad` for 1D elements (see also `-showelt1d`).

**Reply by sajutabraham** [reply]
Thank you very much
